rpm/etcd3-gateway.spec

```diff
@@ -0,0 +1,45 @@
+%{!?python_sitelib: %define python_sitelib %(%{__python} -c "from distutils.sysconfig import get_python_lib; print get_python_lib()")}
+
+Name:           etcd3-gateway
```

Comment:
python-$MODULE is recommended naming for RPM Python modules
https://docs.fedoraproject.org/en-US/packaging-guidelines/Naming/#_python_modules